integration_tests/test_competition_download.py

```diff
@@ -23,6 +23,16 @@ def test_competition_succeeds(self) -> None:
 
             assert_files(self, actual_path, expected_files)
 
+    def test_competition_single_file_succeeds(self) -> None:
```

Comment:
We had a test for downloading all the competition files but none for downloading a single file. They use different handlers so good to have an integration test to prevent any regression.